docs/openapi/components/schemas/common/AgGradeInspectionDefect.yml

```diff
@@ -0,0 +1,61 @@
+$linkedData:
+  term: AgGradeInspectionDefect
+  '@id': https://w3id.org/traceability#AgGradeInspectionDefect
+title: Agriculture Grade Inspection Defect
+description: Information on a type of offsize / defect observed.
+type: object
+properties:
+  type:
+    oneOf:
+      - type: array
+        items:
+          type: string
+          enum:
+            - AgGradeInspectionDefect
+      - type: string
+        const:
+          - AgGradeInspectionDefect
```

Comment:
@BenjaminMoe is this the right syntax for this? I can never remember.

Reply:
I think the correct syntax is:

```
        const:
          - AgGradeInspectionDefect
```

The way to check is change type from an array to a string and see if you get an error.